{
  if (errorCode != ERRORCODE_SUCCESS) {
    throwExceptionForError(errorCode);
  }
}
